admin/class-admin-init.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+class WPSEO_Admin_Init {
+
+	/**
+	 * @var array
+	 */
+	private $options;
+
+	/**
+	 * Class constructor
+	 */
+	function __construct() {
+	 	$this->options = WPSEO_Options::get_all();
+
+		$GLOBALS['wpseo_admin'] = new WPSEO_Admin;
+
+		$this->load_meta_boxes();
+		$this->load_taxonomy_class();
+		$this->load_admin_page_class();
+		$this->load_yoast_tracking();
+		$this->load_tour();
+		$this->load_xml_sitemaps_admin();
+	}
+
+	private function load_meta_boxes() {
+		global $pagenow;
```

Comment:
You could get rid of all the `global $pagenow`'s by assigning `$pagenow` to a property in the constructor and using the property instead.